auth/authentication.go

```diff
@@ -35,8 +35,9 @@ func (a *Auth) RequireAdmin() gin.HandlerFunc {
 		if user != nil {
 			return true, user.Admin, user.ID
 		}
-		if token := a.DB.GetClientByToken(tokenID); token != nil {
-			return true, a.DB.GetUserByID(token.UserID).Admin, token.UserID
+		if token, _ := a.DB.GetClientByToken(tokenID); token != nil {
```

Comment:
This error should also be returned. Same for the other ones in this file.

Reply:
I don't think we should return error during user authentication process, as it might contain sensitive information. Or maybe we can return a general 500 error?

Reply:
yeah, a general error should be enough